overlord/snapstate/snapstate.go

```diff
@@ -3142,7 +3142,10 @@ func removeInactiveRevision(st *state.State, name, snapID string, revision snap.
 
 // RemoveMany removes everything from the given list of names.
 // Note that the state must be locked by the caller.
-func RemoveMany(st *state.State, names []string) ([]string, []*state.TaskSet, error) {
+func RemoveMany(st *state.State, names []string, flags *RemoveFlags) ([]string, []*state.TaskSet, error) {
+	if flags == nil {
```

Comment:
If `RemoveMany` is not actually using the `flags` parameter, then I don't think this check should be in this function. I don't see it as the responsibility of `RemoveMany` to make sure this is non-nil. Instead it's the responsibility of `removeTasks`, where it's used.

Reply:
Removed the check. Thanks